### PR TITLE
Update to iqm-client 4.2 and replace settings file

### PR DESCRIPTION
**Pull request by cqc-alec**
I tried re-enabling the remote tests but they are still failing, now with errors like:
```
>       res = b.run_circuit(c, n_shots=n_shots, timeout=30)

backend_test.py:49: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../env/lib/python3.9/site-packages/pytket/backends/backend.py:384: in run_circuit
    return self.run_circuits(
../env/lib/python3.9/site-packages/pytket/backends/backend.py:409: in run_circuits
    results = self.get_results(handles, **kwargs)
../env/lib/python3.9/site-packages/pytket/backends/backend.py:352: in get_results
    return [self.get_result(handle, **kwargs) for handle in handles]
../env/lib/python3.9/site-packages/pytket/backends/backend.py:352: in <listcomp>
    return [self.get_result(handle, **kwargs) for handle in handles]
../pytket/extensions/iqm/backends/iqm.py:294: in get_result
    self._client.wait_for_results(run_id, timeout_secs=timeout)
../env/lib/python3.9/site-packages/iqm_client/iqm_client.py:570: in wait_for_results
    results = self.get_run(job_id)
../env/lib/python3.9/site-packages/iqm_client/iqm_client.py:522: in get_run
    result = RunResult.from_dict(result.json())
../env/lib/python3.9/site-packages/iqm_client/iqm_client.py:268: in from_dict
    return RunResult(status=Status(input_copy.pop('status')), **input_copy)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   pydantic.error_wrappers.ValidationError: 1 validation error for RunResult
E   metadata
E     field required (type=value_error.missing)

pydantic/main.py:341: ValidationError
```

### CODE REVIEW

File: docs/changelog.rst
Comment (cqc-melf) [inline docs/changelog.rst:4]:
```suggestion
0.5.0 (unreleased)
------------------
```

**Review by cqc-melf — COMMENTED** [review]
Will do a proper review tomorrow

**Review by cqc-melf — APPROVED** [review]
From my side this can be merged now, not sure if we should wait for external feedback?

**Comment by cqc-alec** [comment]
> From my side this can be merged now, not sure if we should wait for external feedback?

OK thanks, will merge this. Keeping #2 open.

**Comment by cqc-melf** [comment]
I still would wait to release this until the remote checks are working again?

**Pull request merged**
